modules/dnn/src/torch/THFile.h

```diff
@@ -28,15 +28,15 @@ TH_API unsigned char THFile_readByteScalar(THFile *self);
 TH_API char THFile_readCharScalar(THFile *self);
 TH_API short THFile_readShortScalar(THFile *self);
 TH_API int THFile_readIntScalar(THFile *self);
-TH_API long THFile_readLongScalar(THFile *self);
+TH_API int64 THFile_readLongScalar(THFile *self);
```

Comment:
Looks like you have to `#include "opencv2/core/hal/interface.h"` to use `int64` type.